tutorials/audio/text_to_speech.rst

```diff
@@ -72,6 +81,12 @@ Distro-specific one-liners
 |                  |     pacman -S speech-dispatcher festival espeakup                                                         |
 +------------------+-----------------------------------------------------------------------------------------------------------+
 
+Troubleshooting
+---------------
+If you get the error `Invalid get index '0' (on base: 'PackedStringArray').` for the line `var voice_id = voices[0]`, check if there are any items in `voices`. If not:
+
+- All users: make sure you enabled Text to Speech in project settings
```

Comment:
```suggestion
- All users: make sure you enabled **Text to Speech** in project settings
```

Reply:
Thanks for this. Sorry, I wasn't sure if the standard was quotes, code backticks, or something else. Will keep it in mind for next time.